ts/components/leftpane/LeftPane.tsx

```diff
@@ -43,33 +33,31 @@ const InnerLeftPaneMessageSection = () => {
   );
 };
 
-const InnerLeftPaneContactSection = () => {
-  return <LeftPaneContactSection />;
-};
-
 const LeftPaneSection = () => {
   const focusedSection = useSelector(getFocusedSection);
 
   if (focusedSection === SectionType.Message) {
     return <InnerLeftPaneMessageSection />;
   }
 
-  if (focusedSection === SectionType.Contact) {
-    return <InnerLeftPaneContactSection />;
-  }
   if (focusedSection === SectionType.Settings) {
     return <LeftPaneSettingSection />;
   }
   return null;
 };
 
+export const leftPaneListWidth = 300;
+
 export const LeftPane = () => {
   return (
     <SessionTheme>
       <div className="module-left-pane-session">
         <ActionsPanel />
 
-        <div className="module-left-pane">
+        <div
+          className="module-left-pane"
+          style={{ '--left-pane-list-width': `${leftPaneListWidth}px` } as CSSProperties}
```

Comment:
because it is specific to the left pane list and not global? I don't think adding every single height/width to the html style is useful if used only in a subtree of it, right?